src/user.ts

```diff
@@ -75,6 +75,14 @@ async function retrieveUserFields(logger: Logger, username: string): Promise<Use
     authInfo: await AuthInfo.create({ username }),
   });
 
+  if (sfdc.matchesAccessToken(username)) {
+    logger.debug('received an accessToken for the username.  Converting...');
+    username = (await connection.identity()).username;
+    logger.debug(`accessToken converted to ${username}`);
+  } else {
+    logger.debug('not a accessToken');
+  }
+
   const fromFields = Object.keys(REQUIRED_FIELDS).map(upperFirst);
```

Comment:
Not part of this change, but I do not see the sense in the transformation of required fields to name case.

@shetzel, is there a reason for this?

Reply:
There was a reason for it at the time.  I'm guessing it was something to do with the User API.